code/modules/mob/say.dm

```diff
@@ -40,6 +40,40 @@
 	say(message, language = language)
 
 
+/mob/living/verb/subtler(message as text)
+
+	set name = "Subtler"
+	set category = "IC"
+	set desc = "Send a 1-tile range message that non-admin ghosts can't see."
+
+	set instant = TRUE
```

Comment:
This isn't set for subtle- also, is this necessary? On tgcode most verbs appear to lack this?

Reply:
To my knowledge it's not necessary, though I do believe set instant = true is a thing on normal says / talks. 
Consider it an artifact of the code.